recipe-client-addon/lib/Storage.jsm

```diff
@@ -29,6 +30,34 @@ function loadStorage() {
 }
 
 this.Storage = {
+  DURABILITY_NAMESPACE: '_storageDurability',
+  DURABILITY_KEY: 'durable',
+  async seedDurability(sandbox) {
+    const globalDurabilityStore = Storage.makeStorage(this.DURABILITY_NAMESPACE, sandbox);
```

Comment:
👍 to both of these ideas. I think the key `_storageDurability` is unique enough and obvious enough what it is doing. I mostly wanted to make sure we were doing this on purpose, knowing the tradeoffs.

As for `prefix`/`namespace`: that's also a good idea. You're right that `prefix` is more a implementation detail, and `namespace` is really the concept we should be conveying. That change would be good in another PR, and I think we'll have to update actions as well.